ox-textile.el

```diff
@@ -147,9 +147,12 @@ contextual information."
   contents)
 
 (defun org-textile-item-list-depth (item info)
-  (let* ((headline-level
-	  (org-export-get-relative-level
-	   (org-export-get-parent-headline item) info))
+  (let* ((headline (org-export-get-parent-headline item))
+	 (headline-level
+	  (if headline
```

Comment:
Since you are handling "no headline" case here, would you mind to add another test for that?

Reply:
I just checked - the "ordered-list" and "unordered-list" tests already fail without this fix since they test cases where no headline is included.